src/api-collector/main.go

```diff
@@ -153,48 +148,82 @@ func getAPISpecsUrls(ctx context.Context, client *github.Client, owner string, r
 	return m.OpenApiSpecs, nil
 }
 
-func downloadAPISpecs(repo string, specsUrls []string) []string {
-	var downloadedSpecs []string
-	for _, url := range specsUrls {
-		resp, err := http.Get(url)
-		if err != nil {
-			log.Printf("Error downloading API spec file: %v\n", err)
-			continue
-		}
-		defer resp.Body.Close()
-
-		if resp.StatusCode != http.StatusOK {
-			log.Printf("Bad HTTP status: %s\n", resp.Status)
-			continue
-		}
-		content, err := io.ReadAll(resp.Body)
+func downloadAPISpecs(ctx context.Context, client *github.Client, owner string, repo string) []string {
+	content, err := getAPISpecFromDir(ctx, client, owner, repo)
```

Comment:
wouldn't this be better to be read as a fallback in case no api are specified in the `.tractusx` file?
this would give the possibility to the project to have the openAPI file in that position aligned with main branch and have the one related to the latest version being in another branch.